CRITICAL - 2022-06-13 13:57:27 --> The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(219): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(177): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\lab11_php_ci\ci4\system\bootstrap.php(146): CodeIgniter\CodeIgniter->initialize()
#3 C:\xampp\htdocs\lab11_php_ci\ci4\spark(45): require('C:\\xampp\\htdocs...')
#4 {main}
ERROR - 2022-06-13 07:24:39 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-06-13 07:24:39 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\lab11_php_ci\ci4\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lab11_php_ci\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lab11_php_ci\ci4\app\Controllers\Artikel.php(12): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(832): App\Controllers\Artikel->index()
#6 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#7 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\lab11_php_ci\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\lab11_php_ci\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-06-13 07:25:49 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-06-13 07:25:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\lab11_php_ci\ci4\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lab11_php_ci\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lab11_php_ci\ci4\app\Controllers\Artikel.php(12): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(832): App\Controllers\Artikel->index()
#6 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#7 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\lab11_php_ci\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\lab11_php_ci\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-06-13 07:25:55 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-06-13 07:25:55 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\lab11_php_ci\ci4\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lab11_php_ci\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lab11_php_ci\ci4\app\Controllers\Artikel.php(12): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(832): App\Controllers\Artikel->index()
#6 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#7 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\lab11_php_ci\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\lab11_php_ci\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-06-13 08:08:03 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-06-13 08:08:03 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\lab11_php_ci\ci4\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lab11_php_ci\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lab11_php_ci\ci4\app\Controllers\Artikel.php(12): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(832): App\Controllers\Artikel->index()
#6 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#7 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\lab11_php_ci\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\lab11_php_ci\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-06-13 08:08:08 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-06-13 08:08:08 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\lab11_php_ci\ci4\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lab11_php_ci\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lab11_php_ci\ci4\app\Controllers\Artikel.php(12): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(832): App\Controllers\Artikel->index()
#6 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#7 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\lab11_php_ci\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\lab11_php_ci\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-06-13 08:10:02 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-06-13 08:10:02 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\lab11_php_ci\ci4\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lab11_php_ci\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lab11_php_ci\ci4\app\Controllers\Artikel.php(12): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(832): App\Controllers\Artikel->index()
#6 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#7 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\lab11_php_ci\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\lab11_php_ci\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-06-13 08:13:46 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-06-13 08:13:46 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\lab11_php_ci\ci4\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lab11_php_ci\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lab11_php_ci\ci4\app\Controllers\Artikel.php(12): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(832): App\Controllers\Artikel->index()
#6 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#7 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\lab11_php_ci\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\lab11_php_ci\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-06-13 08:28:56 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-06-13 08:28:56 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\lab11_php_ci\ci4\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lab11_php_ci\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lab11_php_ci\ci4\app\Controllers\Artikel.php(12): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(832): App\Controllers\Artikel->index()
#6 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#7 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\lab11_php_ci\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\lab11_php_ci\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-06-13 08:30:55 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-06-13 08:30:55 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\lab11_php_ci\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\lab11_php_ci\ci4\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\lab11_php_ci\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\lab11_php_ci\ci4\app\Controllers\Artikel.php(12): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(832): App\Controllers\Artikel->index()
#6 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#7 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\lab11_php_ci\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\lab11_php_ci\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-06-13 08:32:52 --> Invalid file: artikel/index.php
#0 C:\xampp\htdocs\lab11_php_ci\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('artikel/index.p...')
#1 C:\xampp\htdocs\lab11_php_ci\ci4\system\Common.php(1089): CodeIgniter\View\View->render('artikel/index', Array, true)
#2 C:\xampp\htdocs\lab11_php_ci\ci4\app\Controllers\Artikel.php(13): view('artikel/index', Array)
#3 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(832): App\Controllers\Artikel->index()
#4 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#5 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\lab11_php_ci\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\lab11_php_ci\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-06-13 08:32:53 --> Invalid file: artikel/index.php
#0 C:\xampp\htdocs\lab11_php_ci\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('artikel/index.p...')
#1 C:\xampp\htdocs\lab11_php_ci\ci4\system\Common.php(1089): CodeIgniter\View\View->render('artikel/index', Array, true)
#2 C:\xampp\htdocs\lab11_php_ci\ci4\app\Controllers\Artikel.php(13): view('artikel/index', Array)
#3 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(832): App\Controllers\Artikel->index()
#4 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#5 C:\xampp\htdocs\lab11_php_ci\ci4\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\lab11_php_ci\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\lab11_php_ci\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
